github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-postsubmits.yaml

```diff
@@ -0,0 +1,59 @@
+postsubmits:
+  k8snetworkplumbingwg/kubemacpool:
+    - name: main-kubemacpool
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-kubevirtci-quay-credential: "true"
+      branches:
+        - main
+      cluster: phx-prow
+      spec:
+        nodeSelector:
+          type: vm
+          zone: ci
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20201119-a5880e0
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/bash"
+              - "-c"
+              - "cat $QUAY_PASSWORD | docker login --username $(cat $QUAY_USER) --password-stdin=true quay.io && make container docker-push"
+            # docker-in-docker needs privileged mode
+            securityContext:
+              privileged: true
+    - name: release-kubemacpool
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-kubevirtci-quay-credential: "true"
+      branches:
+        - ^v\d+\.\d+\.\d+$
+      cluster: phx-prow
+      spec:
+        nodeSelector:
+          type: vm
+          zone: ci
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20201119-a5880e0
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/bash"
+              - "-c"
+              - |
+                cat $QUAY_PASSWORD | docker login --username $(cat $QUAY_USER) --password-stdin=true quay.io &&
+                # Only push images on tags
+                COMMIT_TAG=$(git tag --points-at HEAD | head -1)
```

Comment:
Maybe we can use one of the prow env vars instead of this ? https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables

Reply:
well the only one that could be relevant is `PULL_REFS` but I'm not sure how I can make sure it holds the correct value (i.e the tag), since it is not used in this repo..